taskcluster/ac_taskgraph/transforms/signing.py

```diff
@@ -0,0 +1,81 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""
+Apply some defaults and minor modifications to the jobs defined in the build
```

Comment:
Eh, that's a bad copy and paste. I'm gonna remove these docstrings for now because I don't think they provide a lot of meaningful context. Please let me know if you want them back, I'll add them in a followup.